public/index.html

```diff
@@ -56,7 +56,18 @@ <h3>Queue batches of lightning channels to open in a single transaction</h3>
 							<button type="button" class="is-circle colored-button" onclick="remove_channel()">-</button>
 						</x-flex>
 					</p>
+					<div>
 				</fieldset>
+				<fieldset>
+					<legend>Recommended Outbound Channels</legend>
```

Comment:
What if we used a verb instead to tell the user to take an action? e.g. "Connect to a suggestion"

Reply:
not a bad idea